### DISCUSSION TITLE
Bug and Discussion: With suspendTabsUntilReady not being reliable and uBlock not filtering reliably on startup, we seriously wonder about your unwillingness to implement Discard, and I seriously wonder about your true intentions and entanglements now.

### DISCUSSION THREAD

**Discussion by Maxxee** [uBlockOrigin/uBlock-issues#3170] [General]
### Prerequisites

- [x] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

With suspendTabsUntilReady not being reliable and uBlock not filtering reliably on startup, we seriously wonder about your unwillingness to implement Discard, and I seriously wonder about your true intentions and entanglements now.

Let's discuss this fact.

### A specific URL where the issue occurs.

```yaml
https://github.com/uBlockOrigin/uBlock-issues/issues/new?assignees=&labels=&projects=&template=bug_report.yml
```

### Steps to Reproduce

.

### Expected behavior

.

### Actual behavior

.

### uBO version

1.56.0

### Browser name and version

Brave

### Operating System and version

Linux

**Comment by gorhill** [comment]
> Let's discuss this fact.

I have no interest to discuss your wonderings.

**Reply by garry-ut99** [reply]
Anyway, it seems OP is speaking about this:
- https://github.com/gorhill/uBlock/issues/1327#issuecomment-353877683

**Comment by uBlock-user** [comment]
@Maxxee I have locked https://github.com/uBlockOrigin/uBlock-issues/issues/3171, do not bother to open another issue. Respond here

**Reply by ghost** [reply]
Blocked.
